### ISSUE TITLE
Personal Tax Return FAQs

### ISSUE THREAD

**Issue by rub1e** [dwyl/UK-HMRC-FAQs#14]
We've had some queries about information needed for personal tax returns (due end January)

* https://github.com/dwyl/hq/issues/512
* https://github.com/dwyl/hq/issues/513

If there are other queries / info people need, now is an excellent time to ask (especially as I'll be filing my own tax return shortly).

So please add any questions you have as comments, and then I'll collate them into a README

**Comment by rub1e** [comment]
@RobStallion thoughtlessly bothered me at home with the following (🙄):

https://www.gov.uk/guidance/tax-free-allowances-on-property-and-trading-income

### DISCLAIMER: I am not qualified to give financial advice and this shouldn't be considered as such. This is MY understanding of part of the personal tax return process, and the analysis is only applicable to ME. You are responsible for ensuring your submissions to HMRC are correct.

> You can get up to £1,000 each tax year in tax-free allowances for property or trading income from 6 April 2017. If you have both types of income, you’ll get a £1,000 allowance for each.

This makes *much* more sense when you fill in your actual return (and I'll try to screenshot the relevant part when I do mine this week), but basically you'd have two options for dealing with your freelance expenses (you can't do both)

* Use your actual costs (only if they're more than £1k, example B)
* Use the flat rate allowance instead of your actual costs (if they're £1k or less, example A)

**Example A**
> Dured is a self-employed photographer. He earns £1,750 income a year from his business and incurs expenses of £600. Using his actual expenses figure would make his profit £1,750 - £600 = £1,150. Using the trading allowance instead of actual expenses would make his profit £750. In order to make his profit lower and save tax, Dured uses the trading allowance to work out his profit.

**Example B**
> Ross is a self-employed graphic designer. He earns £1,750 income a year from his business and incurs expenses of £1,100. Using his actual expenses figure would make his profit £1,750 - £1,100 = £650. Using the trading allowance instead of actual expenses would make his profit £750. In order to make his profit lower and save tax, Ross uses his actual costs to work out his profit, and does not use the trading allowance.

(Examples lifted from [here](https://www.freeagent.com/glossary/trading-allowance/))

It's very unlikely any dwyler will incur expenses of more than £1k (at least not related to dwyl work), so _by far_ the easiest (and cheapest) thing to do is tick the box that says "claim £1k personal allowance" (screenshot to follow) and then not fill in any of the other expenses inputs.

**Comment by iteles** [comment]
@rub1e Your sarcasm really doesn't translate 😆 Also, I would advise that ALL of these issues (especially your last paragraph) should come with a disclaimer that you/we are not qualified to give financial advice and are simply documenting factual information for use by the end user as he/she so pleases

@RobStallion thanks for the info! 🙌

**Comment by rub1e** [comment]
LOL yeah - I think I used an `h1` to say that when I did the pensions issue. Updated.

PS I think my _friend_ @RobStallion knows the score. And I'm _glad_ he feels comfortable bothering me at home on the weekend with work questions. (Better than some of the other stuff he sends me 😱)

**Comment by iteles** [comment]
Indeed. But you've gotta remember these issues are intended to be useful to the whole dwyl community and are not just read by your _friends_! 💁

**Comment by rub1e** [comment]
Indeed - but the whole dwyl community should not feel free to bother me at home 😉

**Comment by rub1e** [comment]
Here is the screenshot of the relevant section for the tax-free allowance on trading, along with HMRC's much clearer narrative of how to claim it.

For me, in this tax period, I had one self-employment and less than £1000 in expenses, so I've claimed the full £1000 allowance for that one business. (This is pretty much the simplest case, and I'd imagine the most common)

![untitled presentation](https://user-images.githubusercontent.com/11595920/50959988-38270800-14bc-11e9-98fb-b93430dc82c0.png)

And so since I've claimed the flat rate of £1000 for the allowance, I'm not allowed to input my _actual_ expenses in the next section, so the expenses box is left blank:

![selection_396](https://user-images.githubusercontent.com/11595920/50960177-a8358e00-14bc-11e9-811f-a016728d63e3.png)
